            void assertInProgress() throws Exception
            {
                assertFiles(txnLogs.getDataFolder(), Sets.newHashSet(Iterables.concat(sstableNew.getAllFilePaths(),
                                                                                      sstableOld.getAllFilePaths(),
                                                                                      Collections.singleton(txnLogs.getLogFile().file.getPath()))));
            }

